src/corona_hakab_model/policies_manager.py

```diff
@@ -9,13 +9,18 @@ def __init__(self, manager: "SimulationManager"):
         self.update_matrix_manager = manager.update_matrix_manager
         self.consts = manager.consts
         self.logger = manager.logger
+        # For each applied policy, save a list of affected circles
+        self.daily_affected_circles: Dict[ConditionedPolicy, List[SocialCircle]] = None
```

Comment:
Wouldn't it be simpler to just initialize it to a `defaultdict(List)`, and empty it at the beginning of every `perform_policies` step?

Reply:
I wanted to separate the cases of "no policy was activated" and "some policies were activated, but none of the circles were affected". These two cases can't be separated if this variable contains an empty dictionary.